lib/screens/group_member_list/widgets/group_member_tile.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:transparent_image/transparent_image.dart';
+
+///This class returns a list tile which would hold the name and details of member
+///It receives a 'title' as a required named argument which is the name of the member and url of profile image as a named argument
+
+class GroupMemberTile extends StatelessWidget {
+  final String title, profilePicUrl;
+
+  GroupMemberTile({@required this.title, this.profilePicUrl});
```

Comment:
sorry to mention it late you can take in member id also :sweat_smile:

Reply:
I assume that member ID is going to be an integer and so I have added functionalities accordingly sir.